MatomoTracker.php

```diff
@@ -1477,6 +1477,19 @@ public function setRequestTimeout($timeout)
         $this->requestTimeout = $timeout;
         return $this;
     }
+	
+	/**
+     * Sets the request method (either GET or POST). POST is recommended when using
+     * setTokenAuth() to prevent the token from being recorded in server logs.
```

Comment:
It might be good to add a comment:

```
When using POST, make sure there's no redirect happening on your server as otherwise POSTed values get lost.
```

Reply:
added commit